linode/manager
# Refactor: [M3-6522] - Update Link component export and improve Storybook story

## Description 📝
This PR refactors the `Link` component exports and handles the new way of doing stories. It does not modify the component in any way, or at least shouldn't. Part of our epic to migrate away from .mdx files.

## Preview 📷
No visual diff

## How to test 🧪
1. Confirm no regression on the Link component
2. Confirm new story at /?path=/docs/components-link--documentation
